src/Microsoft.IdentityModel.JsonWebTokens/JwtHeaderParameterNames.cs

```diff
@@ -28,69 +28,69 @@
 namespace Microsoft.IdentityModel.JsonWebTokens
 {
     /// <summary>
-    /// List of header parameter names see: http://tools.ietf.org/html/rfc7519#section-5.
+    /// List of header parameter names see: https://datatracker.ietf.org/doc/html/rfc7519#section-5.
     /// </summary>
     public struct JwtHeaderParameterNames
     {
         /// <summary>
-        /// see:https://tools.ietf.org/html/rfc7515#section-4.1.1
+        /// See: https://datatracker.ietf.org/doc/html/rfc7515#section-4.1.1
         /// </summary>
         public const string Alg = "alg";
 
         /// <summary>
-        /// see:https://tools.ietf.org/html/rfc7515#section-4.1.10
-        /// also:https://tools.ietf.org/html/rfc7519#section-5.2
+        /// See: https://datatracker.ietf.org/doc/html/rfc7515#section-4.1.10
+        /// also:https://datatracker.ietf.org/doc/html/rfc7519#section-5.2
         /// </summary>
         public const string Cty = "cty";
 
         /// <summary>
-        /// see:https://tools.ietf.org/html/rfc7516#section-4.1.2
+        /// See: https://datatracker.ietf.org/doc/html/rfc7516#section-4.1.2
         /// </summary>
         public const string Enc = "enc";
 
         /// <summary>
-        /// see:https://tools.ietf.org/html/rfc7518#section-4.7.1.1
+        /// See: https://datatracker.ietf.org/doc/html/rfc7518#section-4.7.1.1
         /// </summary>
         public const string IV = "iv";
 
         /// <summary>
-        /// see:https://tools.ietf.org/html/rfc7515#section-4.1.2
+        /// see:https://datatracker.ietf.org/doc/html/rfc7515#section-4.1.2
         /// </summary>
         public const string Jku = "jku";
 
         /// <summary>
-        /// see:https://tools.ietf.org/html/rfc7515#section-4.1.3
+        /// See: https://datatracker.ietf.org/doc/html/rfc7515#section-4.1.3
         /// </summary>
         public const string Jwk = "jwk";
 
         /// <summary>
-        /// see:https://tools.ietf.org/html/rfc7515#section-4.1.4
+        /// See: https://datatracker.ietf.org/doc/html/rfc7515#section-4.1.4
         /// </summary>
         public const string Kid = "kid";
 
         /// <summary>
-        /// see:https://tools.ietf.org/html/rfc7515#section-4.1.9
-        /// also:https://tools.ietf.org/html/rfc7519#section-5.1
+        /// See: https://datatracker.ietf.org/doc/html/rfc7515#section-4.1.9
+        /// also:https://datatracker.ietf.org/doc/html/rfc7519#section-5.1
```

Comment:
For consistency, also -> Also

Reply:
changed in e89d4fbafb9ac32deb2b8b8a704e473b9e647669